### ISSUE TITLE
Installing a local extension causes it to get cached, and it can't be changed and re-installed again

### ISSUE THREAD

**Issue by Ruin0x11** [standardnotes/desktop#657]
I was trying to follow the steps at [Local Setup](https://docs.standardnotes.org/extensions/local-setup/) to install a new theme. I ran `http-server` in a theme extension's directory and installed it with the URL `http://localhost:8001/ext.json`. This is on Windows. Then I changed some things and uninstalled the extension. When I re-installed the extension, the changes were not reflected in the application. The only way to fix this was to change the extension `identifier` every time I make a change. It's confusing since otherwise it looks like the install succeeds.

**Comment by johnny243** [comment]
Yes @Ruin0x11, the extensions are cached. You can open DevTools, then go to the **Network** tab and make sure "Disable cache" is checked. Your extension should not be cached as long as DevTools is open.

**Comment by Ruin0x11** [comment]
@johnny243 Thanks for clarifying. Can this be added to the docs so nobody else gets caught by this? (https://docs.standardnotes.org/extensions/local-setup)
